recipes/nvitop/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "nvitop" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nvitop-{{ version }}.tar.gz
+  sha256: 3d9be8b355fae4960e9bfcda6111a0cde601880a4deaf1129e95f84a250d9b4b
+
+build:
+  skip: true  # [py<35]
+  entry_points:
+    - nvitop = nvitop.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - cachetools >=1.0.1
+    - colorama >=0.4.0  # [win]
+    - nvidia-ml-py >=11.450.51,<11.500.00
+    - psutil >=5.6.6
+    - python
+    - termcolor >=1.0.0
+    - windows-curses >=2.2.0  # [win]
+
+test:
+  imports:
+    - nvitop
+  commands:
+    - pip check
+    - nvitop --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/XuehaiPan/nvitop
+  summary: An interactive NVIDIA-GPU process viewer, the one-stop solution for GPU process management.
+  dev_url: https://github.com/XuehaiPan/nvitop
+  license: GPL-2.0-or-later
```

Definitely not 2, but 3. Also seems like 3 only?
```suggestion
  license: GPL-3.0-only
```